mmengine/hooks/sync_buffer_hook.py

```diff
@@ -22,3 +40,15 @@ def after_train_epoch(self, runner) -> None:
         """
         if self.distributed:
             all_reduce_params(runner.model.buffers(), op='mean')
+            self.called_in_train = True
+
+    def before_test_epoch(self, runner) -> None:
```

Comment:
Sorry for my missing details. This method is not necessary to be implemented because `after_train_epoch` would sync the buffers for both `IterBasedTrainLoop` and `EpochBasedTrainLoop`.

Reply:
Do you mean that `before_test_epoch` is not neccessary?

Reply:
Yes